src/components/CustomTable/StyledTableCell/StyledTableCell.js

```diff
@@ -0,0 +1,24 @@
+import { TableCell, withStyles } from "@material-ui/core"
+
+import replaceIfNull from "@/utils/replaceIfNull"
+
+const NewTableCell = withStyles((theme) => ({
+  root: {
+    padding: "10px",
+  },
+  head: {
+    backgroundColor: theme.palette.primary.light,
+    color: theme.palette.secondary.dark,
+    fontWeight: theme.typography.fontWeightBold,
+  },
+  body: {
+    borderBottomColor: theme.palette.primary.dark,
+    textTransform: "capitalize",
+  },
+}))(TableCell)
+
+const StyledTableCell = ({ children }) => {
+  return <NewTableCell>{replaceIfNull(children)}</NewTableCell>
+}
```

Comment:
I wouldn't actually recommend doing this. `children` isn't limited to a string, it's actually a react node. You should be calling `replaceIfNull` at the point you get the string.